src/components/MessageCard/MessageCard.jsx

```diff
@@ -13,8 +13,8 @@ import { MessageCardContent } from './components/MessageCard.Content'
 import { MessageCardSurvey } from './components/MessageCard.Survey'
 import MessageCardUrlAttachmentImage from './components/MessageCard.UrlAttachmentImage'
 import MessageCardArticleCard from './components/MessageCard.ArticleCard'
-
-function noop() {}
+import { MessageCardVideo } from './components/MessageCard.Video'
+import { noop } from '../../utilities/other'
```

Comment:
`noop` usage in hsds-react coming from 'utils/other' has been removed. The only reason it still lives in the repo is because it's used in apps and we ned to update them to not use it.

Reply:
ah, thanks, good to know!